week06/Number_of_Matching_Subsequences.cpp

```diff
@@ -0,0 +1,32 @@
+// T.C: O(S + W*log(S)) Here, S = number of characters in string, s
+// M.C: O(S)                  W = sum of all characters in strings words, w
+
+
+class Solution {
+public:
+    vector <int> index_of[26];
+    vector <int> :: iterator it;
```

Comment:
Just declare it inside `isMatched)` and make the type `auto` :) (Why make your life difficult)

Reply:
My bad coding habits. This always happen when I get little uncareful :'(